src/Swashbuckle.Swagger/project.json

```diff
@@ -1,27 +1,28 @@
 ﻿{
-  "version": "6.0.0-beta9",
-  "description": "Core components for Swashbuckle - a Swagger toolchain for AspNet Core",
-  "authors": [ "Richard Morris" ],
-
-  "packOptions": {
-    "tags": [ "Swagger Documentation Discovery Help WebApi AspNet AspNetCore" ],
-    "projectUrl": "https://github.com/domaindrivendev/Swashbuckle",
-    "licenseUrl": "https://github.com/domaindrivendev/Swashbuckle/LICENSE",
-    "summary": "Provides a Swagger object model and middleware that can be used to expose Swagger descriptions for your API"
-  },
-
   "dependencies": {
-    "Microsoft.AspNetCore.Mvc.Core": "1.0.0-rc2-final",
-    "Microsoft.AspNetCore.Mvc.Formatters.Json": "1.0.0-rc2-final"
+    "Microsoft.AspNetCore.Mvc.Core": "1.0.0",
+    "Microsoft.AspNetCore.Mvc.Formatters.Json": "1.0.0"
   },
-
+  "description": "Core components for Swashbuckle - a Swagger toolchain for AspNet Core",
   "frameworks": {
-    "netstandard1.5": {
+    "net451": {},
+    "netstandard1.6": {
       "imports": [
         "dnxcore50",
         "portable-net451+win8"
       ]
-    },
-    "net451": { }
-  }
+    }
+  },
+  "packOptions": {
+    "licenseUrl": "https://github.com/domaindrivendev/Swashbuckle/LICENSE",
+    "owners": [
+      "Richard Morris"
+    ],
+    "projectUrl": "https://github.com/domaindrivendev/Swashbuckle",
+    "summary": "Provides a Swagger object model and middleware that can be used to expose Swagger descriptions for your API",
+    "tags": [
+      "Swagger Documentation Discovery Help WebApi AspNet AspNetCore"
+    ]
+  },
+  "version": "6.0.0-rtm1"
```

Comment:
Unfortunately it can't be `beta10`. The problem is that NuGet uses the suffix _alphabetically_ to determine what's new. So, alphabetically, `beta10` is _older_ than `beta9`. That's actually why they switched early on from releasing `-rc2` style suffixes to `-rc2-final` - because people using early drops couldn't _upgrade_ to an _older_ version (`-rc2` < `-rc2-12345` < `-rc2-final`). When I have CI hooked up, I usually use the build number (~3 digits) as the beta suffix so I don't run into this issue.

I arbitrarily picked `rtm` so when I published to our internal NuGet server it would be seen as newer. For now I'll switch to `-rc-001` so it'll work with NuGet but won't be seen as RTM ready.